### ISSUE TITLE
The plugin is not compatible with latest pytlint/astroid

### ISSUE THREAD

**Issue by datoux** [pexip/pylint-forbidden-imports#8]
Astroid module has removed the AstroidBuildingExeception in favor of AstroidBuildingError:
https://github.com/pylint-dev/astroid/blob/5b665e7e760a7181625a24b3635e9fec7b174d87/ChangeLog#L111

Currently the plugin fails with this stack trace:
```
Traceback (most recent call last):
  File "/Users/dturecek/Library/Caches/pypoetry/virtualenvs/krypton-kag-Pvfaqf7E-py3.12/lib/python3.12/site-packages/pylint/lint/pylinter.py", line 787, in _lint_file
    check_astroid_module(module)
  File "/Users/dturecek/Library/Caches/pypoetry/virtualenvs/krypton-kag-Pvfaqf7E-py3.12/lib/python3.12/site-packages/pylint/lint/pylinter.py", line 1016, in check_astroid_module
    retval = self._check_astroid_module(
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/dturecek/Library/Caches/pypoetry/virtualenvs/krypton-kag-Pvfaqf7E-py3.12/lib/python3.12/site-packages/pylint/lint/pylinter.py", line 1068, in _check_astroid_module
    walker.walk(node)
  File "/Users/dturecek/Library/Caches/pypoetry/virtualenvs/krypton-kag-Pvfaqf7E-py3.12/lib/python3.12/site-packages/pylint/utils/ast_walker.py", line 94, in walk
    self.walk(child)
  File "/Users/dturecek/Library/Caches/pypoetry/virtualenvs/krypton-kag-Pvfaqf7E-py3.12/lib/python3.12/site-packages/pylint/utils/ast_walker.py", line 91, in walk
    callback(astroid)
  File "/Users/dturecek/Library/Caches/pypoetry/virtualenvs/krypton-kag-Pvfaqf7E-py3.12/lib/python3.12/site-packages/pylint_restricted_imports/__init__.py", line 209, in visit_importfrom
    self._check_restricted_imports(node)
  File "/Users/dturecek/Library/Caches/pypoetry/virtualenvs/krypton-kag-Pvfaqf7E-py3.12/lib/python3.12/site-packages/pylint_restricted_imports/__init__.py", line 142, in _check_restricted_imports
    modules = [self._import_module(node, n) for n, _ in node.names]
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/dturecek/Library/Caches/pypoetry/virtualenvs/krypton-kag-Pvfaqf7E-py3.12/lib/python3.12/site-packages/pylint_restricted_imports/__init__.py", line 106, in _import_module
    except AstroidBuildingException:
           ^^^^^^^^^^^^^^^^^^^^^^^^
NameError: name 'AstroidBuildingException' is not defined. Did you mean: 'AstroidBuildingError'?

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/dturecek/Library/Caches/pypoetry/virtualenvs/krypton-kag-Pvfaqf7E-py3.12/lib/python3.12/site-packages/pylint/lint/pylinter.py", line 751, in _lint_files
    self._lint_file(fileitem, module, check_astroid_module)
  File "/Users/dturecek/Library/Caches/pypoetry/virtualenvs/krypton-kag-Pvfaqf7E-py3.12/lib/python3.12/site-packages/pylint/lint/pylinter.py", line 789, in _lint_file
    raise astroid.AstroidError from e
astroid.exceptions.AstroidError

```

**Comment by huwcbjones** [comment]
Fixed in [0.2.1](https://github.com/pexip/pylint-restricted-imports/releases/tag/releases%2F0.2.1)

**Comment by huwcbjones** [comment]
Actually apologies, it appears I forgot to release the new version after we fixed the issues 3 months ago

**Comment by huwcbjones** [comment]
Fixed in aa0baa2e6c77e1a73e7dfa49e84beea5062f447d

**Comment by huwcbjones** [comment]
Fix released in [0.2.2](https://github.com/pexip/pylint-restricted-imports/releases/tag/releases%2F0.2.2), thanks for reporting!

**Comment by datoux** [comment]
Thank you for quick fix !!
